easybuild/tools/options.py

```diff
@@ -681,7 +692,7 @@ def process_software_build_specs(options):
         tryval = getattr(options, 'try_%s' % opt)
         if val or tryval:
             if val and tryval:
-                self.log.warning("Ignoring --try-%(opt)s, only using --%(opt)s specification" % {'opt': opt})
+                logger.warning("Ignoring --try-%(opt)s, only using --%(opt)s specification" % {'opt': opt})
```

Comment:
I'm not saying I was aiming for 100% coverage, I was just surprised we were not testing this code path yet.

Getting this covered by adding two lines to an existing test was fairly trivial (see https://github.com/JensTimmerman/easybuild-framework/issues/9), *and* it revealed another undiscovered bug, so this is fixed now.

Thanks for catching this!